### PR TITLE
chore(docs): add summary table to README.md

### PR DESCRIPTION
**Pull request by toniblyx**
### Context

To help users understand what they will be able to do with Prowler, we have added a table with details of what it is covered so far and what is coming soon.
Screenshot 2023-05-23 at 17 54 48

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,15 @@
...
-It contains hundreds of controls covering CIS, PCI-DSS, ISO27001, GDPR, HIPAA, FFIEC, SOC2, AWS FTR, ENS and custom security frameworks.
+It contains hundreds of controls covering CIS, NIST 800, NIST CSF, CISA, RBI, FedRAMP, PCI-DSS, GDPR, HIPAA, FFIEC, SOC2, GXP, AWS Well-Architected Framework Security Pillar, AWS Foundational Technical Review (FTR), ENS (Spainish National Security Schema) and you custom security frameworks.
+
+| Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.cloud/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.cloud/en/latest/tutorials/misc/#categories) |
+|---|---|---|---|---|
+| AWS | 281 | 54: `prowler aws --list-services` | 21: `prowler aws --list-compliance` | 6: `prowler aws --list-categories` |
+| GCP | 43 | 7: `prowler gcp --list-services` | CIS soon | - `prowler gcp --list-categories`|
+| Azure | 20 | 3: `prowler gcp --list-services` | CIS soon | 1: `prowler azure --list-categories` |
+| Kubernetes | Planned | - | - | - |
+|
```
Comment (MrCloudSec) [inline README.md]:
```suggestion
| Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.cloud/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.cloud/en/latest/tutorials/misc/#categories) |
|---|---|---|---|---|
| AWS | 281 | 54 -> `prowler aws --list-services` | 21 -> `prowler aws --list-compliance` | 6 -> `prowler aws --list-categories` |
| GCP | 43 | 7 -> `prowler gcp --list-services` | CIS soon | 0 -> `prowler gcp --list-categories`|
| Azure | 20 | 3 -> `prowler gcp --list-services` | CIS soon | 1 -> `prowler azure --list-categories` |
| Kubernetes | Planned | - | - | - |
|
```

Comment (MrCloudSec) [reply README.md]: How about putting arrows?

Comment (sergargar) [inline README.md:46]: There is an extra pipe | at the end

Comment (MrCloudSec) [inline README.md]:
```suggestion
| Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.cloud/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.cloud/en/latest/tutorials/misc/#categories) |
|---|---|---|---|---|
| AWS | 281 | 54: `prowler aws --list-services` | 21: `prowler aws --list-compliance` | 6: `prowler aws --list-categories` |
| GCP | 43 | 7: `prowler gcp --list-services` | CIS soon | - `prowler gcp --list-categories`|
| Azure | 20 | 3: `prowler gcp --list-services` | CIS soon | 1: `prowler azure --list-categories` |
| Kubernetes | Planned | - | - | - |
```

**Review by MrCloudSec — CHANGES_REQUESTED** [review]
I love it! Please check my comments.

File: README.md (hunk 1 above)
Comment (MrCloudSec) [reply README.md]: There is an extra pipe | at the end.

Comment (toniblyx) [reply README.md]: No hahaha

Comment (toniblyx) [reply README.md]: needed for the last line of the table

Comment (MrCloudSec) [reply README.md]: It is not needed

**Review by MrCloudSec — DISMISSED** [review]
good job!

File: README.md [hunk 2]
```diff
@@ -35,7 +35,14 @@
 
 `Prowler` is an Open Source security tool to perform AWS, GCP and Azure security best practices assessments, audits, incident response, continuous monitoring, hardening and forensics readiness.
 
-It contains hundreds of controls covering CIS, PCI-DSS, ISO27001, GDPR, HIPAA, FFIEC, SOC2, AWS FTR, ENS and custom security frameworks.
+It contains hundreds of controls covering CIS, NIST 800, NIST CSF, CISA, RBI, FedRAMP, PCI-DSS, GDPR, HIPAA, FFIEC, SOC2, GXP, AWS Well-Architected Framework Security Pillar, AWS Foundational Technical Review (FTR), ENS (Spainish National Security Schema) and you custom security frameworks.
+
+| Provider | Checks | Services | [Compliance Frameworks](https://docs.prowler.cloud/en/latest/tutorials/compliance/) | [Categories](https://docs.prowler.cloud/en/latest/tutorials/misc/#categories) |
+|---|---|---|---|---|
+| AWS | 281 | 54 -> `prowler aws --list-services` | 21 -> `prowler aws --list-compliance` | 6 -> `prowler aws --list-categories` |
+| GCP | 43 | 7 -> `prowler gcp --list-services` | CIS soon | 0 -> `prowler gcp --list-categories`|
```
Comment (MrCloudSec) [inline README.md]:
```suggestion
| GCP | 47 | 7 -> `prowler gcp --list-services` | CIS soon | 0 -> `prowler gcp --list-categories`|
```

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**